src/labeledinput/labeledinput.js

```diff
@@ -0,0 +1,132 @@
+/**
+ * @license Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import uid from '../../utils/uid.js';
+
+import Model from '../model.js';
+import Controller from '../controller.js';
+
+import InputText from '../inputtext/inputtext.js';
+import InputTextView from '../inputtext/inputtextview.js';
+
+import InputLabel from '../inputlabel/inputlabel.js';
+import InputLabelView from '../inputlabel/inputlabelview.js';
+
+/**
+ * The labeled input controller class. It contains two components {@link ui.input.InputLabel InputLabel} and
+ * {@link ui.input.InputText InputText} pair each other by {@link ui.input.LabeledInputModel#uid unique id}.
```

Comment:
"It contains two components, {@link ui.input.InputLabel InputLabel} and {@link ui.input.InputText InputText}, connected by an {@link ui.input.LabeledInputModel#uid unique id}."